### PR TITLE
Fixing ESlint errors for card 225

### LINKED ISSUE #225: Fix the eslint warnings
There are kind of a lot of them.

### PR DESCRIPTION
**Pull request by hannahkamundson**
closes #225

### CODE REVIEW

File: Frontend/src/components/globalStyles/MenuBar.jsx [hunk 1]
```diff
@@ -1,5 +1,4 @@
-import React, { useState } from 'react';
-import styles from './MenuBar.css';
```
Comment (jberns) [inline Frontend/src/components/globalStyles/MenuBar.jsx]: I don't think this was supposed to be deleted

File: Frontend/src/components/globalStyles/Loader.jsx [hunk 2]
```diff
@@ -1,7 +1,6 @@
-import React, { useState } from 'react';
-import styles from './Loader.css';
```
Comment (jberns) [inline Frontend/src/components/globalStyles/Loader.jsx]: I don't think the loader styles were to be deleted

File: Frontend/src/components/globalStyles/MenuBar.jsx (hunk 1 above)
Comment (hannahkamundson) [reply Frontend/src/components/globalStyles/MenuBar.jsx]: Added it back

File: Frontend/src/components/globalStyles/Loader.jsx (hunk 2 above)
Comment (hannahkamundson) [reply Frontend/src/components/globalStyles/Loader.jsx]: you are right. added it back

**Comment by hannahkamundson** [comment]
@divlogic @jberns check it out now. I think I got rid of that error you were talking about @divlogic

**Comment by divlogic** [comment]
It looks like the bug was fixed, but I also think that since master has been pulled into this branch, more eslint warnings have been added. Going to approve this pr though.

**Pull request merged**